intercept_server_test.go

```diff
@@ -182,9 +184,7 @@ func TestInterceptServerStream(t *testing.T) {
 	}
 
 	replies := []interface{}{svr.resp}
-	if !reflect.DeepEqual(replies, str.resps) {
-		t.Fatalf("unexpected reply: expecting %v; got %v", replies, str.resps)
-	}
+	checkProtosEqual(t, replies, str.resps)
```

Comment:
I suppose it was never _guaranteed_ to work for all message types, but it was kind of nice that it used to work. Now we have to do `proto.Equal` to compare them, so that meant updates to these test assertions :/

Reply:
So much yuck